test/smoke_test/bcda_client.go

```diff
@@ -188,9 +190,13 @@ func main() {
 				}
 
 				encryptData := map[string]string{}
-				encOutput := (*json.RawMessage)(objmap["KeyMap"])
-				if err := json.Unmarshal(*encOutput, &encryptData); err != nil {
-					panic(err)
+
+				encryptionEnabled := utils.GetEnvBool("ENABLE_ENCRYPTION", true)
```

Comment:
How/where is this being tested with `ENABLE_ENCRYPTION=false`?

Reply:
I tested this by putting the ENABLE_ENCRYPTION=false in both our docker-compose.yml (for the api responses) and docker-compose.test.yml (for the smoke test).
Tests passed with both the inclusion and exclusion of that feature flag. (Unencrypted and Encrypted respectively). Open to other suggestions though..

Reply:
I wonder if it makes sense to test with encryption off in local/Travis.
When we deploy to `dev`, encryption will be enabled.  We can let that marinade, and then when we're ready, flip the feature flag.  And progressively build up to flipping the flag in all envs.
But I think we'll need to get started in local/Travis, so I think it makes sense to turn it off as part of local environment in this PR.  
Does that make sense, or is there a reason to keep it on in local?

Reply:
With your suggestion, if we _do_ make a breaking change to encryption before it's completely removed, we won't detect it until later in our deployment process.  I support accepting the risk, though, given the short timeline.